src/components/Thread.vue

```diff
@@ -158,6 +172,14 @@ export default {
 		window.removeEventListener('resize', this.resizeDebounced)
 	},
 	methods: {
+		async updateSummary() {
+			if (this.thread.length < 2 && !this.enabledThreadSummary) return
```

Comment:
```suggestion
			if (this.thread.length < 2 || !this.enabledThreadSummary) return
```

Should be an or condition, right? Thread is small or summary is disabled globally.